package.json

```diff
@@ -48,9 +49,11 @@
     "typescript": "^2.1.5"
   },
   "dependencies": {
+    "@types/react-router": "^3.0.6",
     "asap": "https://github.com/alloy/asap.git",
     "react": "^15.4.2",
     "react-relay": "https://github.com/alloy/relay/releases/download/v0.9.3/react-relay-0.9.3.tgz",
+    "react-router": "^3.0.2",
```

Comment:
@alloy I understand your concern about adding complexity. Since the number of pages in this particular app is low and there won't be many route change, I think it's fair to render pages server side without `react-router`. 

I was mainly trying to figure out if we needed to generate separate bundle/entrypoint files for each page. I took a closer look at how webpack bundling is setup in `relational-theory` and we can do the same with fuse-box.